charts/vector/README.md

```diff
@@ -92,7 +92,8 @@ helm install --name <RELEASE_NAME> \
 | dnsConfig | object | `{}` | Specify DNS configuration options for Vector Pods |
 | dnsPolicy | string | `"ClusterFirst"` | Specify DNS policy for Vector Pods |
 | env | list | `[]` | Set environment variables in Vector containers |
-| existingConfigMap | string | `""` | Use existing ConfigMap for Vector's configuration instead of creating a new one |
+| existingConfig | string | `""` | Use this existing ConfigMap for Vector's configuration instead of creating a new one |
```

Comment:
The rename here is for two reasons:

1. Better matching the `extraConfigs`
2. Leaving space for an eventual `pipelines` option.

Both of which are just naming, I couldn't think of something that fit with the existing nicely like `extraConfigConfigMaps` (because one day an `extraPipelinesConfigMaps` will probably be needed). Then the same for eventual `existingPipelineConfigMap` is _fine_ish but leaves `existingConfigMap` more vague and not matching the "naming scheme"